taprpc/taprootassets.proto

```diff
@@ -199,6 +199,9 @@ message AnchorInfo {
     anchor output.
     */
     bytes tapscript_sibling = 7;
+
+    // The height of the block which contains the anchor transaction.
+    uint32 block_height = 8;
```

Comment:
We usually generate the protos in the same commit as we add/update fields.

Reply:
Also applies to the sqlc generate step as well IIRC.